definitions/json/CaseTypeTab/CaseTypeTab-prod.json

```diff
@@ -91,5 +91,14 @@
     "TabID": "initialConsideration",
     "CaseFieldID": "icDateCompleted",
     "TabFieldDisplayOrder": 13
+  },
```

Comment:
Im confused, this looks the same, why did you delete it from prod and move to non prod ?

Reply:
I moved it to prod as per Jack's comment above?

Reply:
That's right, this is the prod file. The object will be different between prod and non prod, so it needs to removed from the base CaseTypeTab.json and added to CaseTypeTab-prod.json (so that we don't lose this in prod) and the modified version in CaseTypeTab-Multiples-nonprod.json